genienlp/tasks/almond.py

```diff
@@ -32,14 +32,14 @@
 import logging
 from collections import defaultdict
 
-from ..base_task import BaseTask
-from ..registry import register_task
-from ..generic_dataset import CQA, default_batch_fn, input_then_output_len, input_tokens_fn
-from ...data_utils.example import Example
-from ...data_utils.progbar import progress_bar
-from .utils import ISO_to_LANG, is_device, is_entity, is_entity_marker, process_id, is_cjk_char
+from .base_task import BaseTask
+from .registry import register_task
+from .generic_dataset import CQA, default_batch_fn, input_then_output_len, input_tokens_fn
+from ..data_utils.example import Example
+from ..data_utils.progbar import progress_bar
+from .utils import is_device, is_entity, is_entity_marker, process_id, is_cjk_char
 
-from ..base_dataset import Split
+from .base_dataset import Split
```

Comment:
Is there a particular reason to move this file? The "utils.py" was meant to be almond-specific utils.

Reply:
No reason at all. Do you want me to move the almond files back to a subfolder?

Reply:
No it's ok, no need to churn on top of churn. It's fine to make the change.

Reply:
While at it can you please rename `genienlp/tasks/utils.py` to ` genienlp/tasks/almond_utils.py` ?